src/fs-app/index.js

```diff
@@ -0,0 +1,111 @@
+const chalk = require('chalk')
+const request = require('request')
+const yargs = require('yargs')
+
+const port = 3000
+const baseUrl = `http://localhost:${port}/fs/`
+
+yargs.command({
+    command: 'get',
+    builder: {
+        name: {
+            demandOption: true,
+            type: 'string'
+        }
+    },
+    handler: function(argv) {
+        let url = baseUrl + argv.name
+        request({ url: url }, (err, res) => {
+            if (err) {
+                console.log(chalk.red(err))
```

Comment:
Create a logger service and use methods from them.

For this application you can wrap log method with code

```
log(logData) {
  console.log(logData);
}
```

We need this because if we plan to change log destination (for example, from console log to to database) we don't need to change all places in the code anymore